src/main/java/org/broadinstitute/consent/http/resources/UserResource.java

```diff
@@ -162,6 +163,23 @@ public Response delete(@PathParam("email") String email, @Context UriInfo info)
         return Response.ok().build();
     }
 
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/SigningOfficials")
```

Comment:
Prefer all lower case urls. Mixed case urls are weirdly supported on windows systems, but since we aren't on windows, it has no practical effect. It is, however, a best practice to use dashes. 
```suggestion
    @Path("/signing-officials")
```

Another implementation approach is to add limited SO info to the `/me` call and not open up a new endpoint. Let's discuss in standup.